Source/Font.swift

```diff
@@ -13,19 +13,39 @@ import UIKit
 public protocol Font: RawRepresentable, Hashable where Self.RawValue == String {
     
     /**
-     Defines a mapping convert one Font weight to another in the case that `UIAccessibility.isBoldTextEnabled` is true. Does not apply to watchOS.
+     Defines a mapping to convert one Font weight to another in the case that `UIAccessibility.isBoldTextEnabled` is true. Does not apply to watchOS.
      
      Example:
      ```
-     static let boldTextMapping: [MyFont: MyFont]? = [
-        .regular: .bold
-     ]
+     var boldTextMapping: MyFont {
+         switch self {
+         case .regular: return .bold
+         case .bold: return .black
+         case .black: return self
+         }
+     }
      ```
      Now every regular `MyFont` instance will become bold if the user has "Bold Text" turned on in their device settings.
      
      If you'd like, you can observe `UIAccessibility.boldTextStatusDidChangeNotification` via `NotificationCenter` and set your fonts when that updates.
      */
-    static var boldTextMapping: [Self: Self]? { get }
+    var boldTextMapping: Self { get }
+    
+    /**
+     Defines a list of fonts to fall back to in the case that characters are used which the font does not support. Uses `UIFontDescriptor.AttributeName.cascadeList`.
+     
+     Example:
+     ```
+     var cascadeList: [CascadingFontProperties] {
+         switch self {
+         case .regular:
```

Comment:
Good addition here, good to add this to the demo project file Shared/Fonts.swift too so no one misses it.